cypress/e2e/attestation.cy.js

```diff
@@ -1,6 +1,7 @@
 describe("Attestation functionality", () => {
   beforeEach(() => {
     cy.useFlag("formTimer", false);
+    cy.useFlag("reCaptcha", false);
```

Comment:
Everything looks good to me.
Just curious but did the global `cy.useFlag("reCaptcha", false)` in `/cypress/support/e2e.js` no longer function?
https://github.com/cds-snc/platform-forms-client/blob/cabea6afeecf01445734e0b6d6c5446b748b8597/cypress/support/e2e.js#L31

Reply:
Pushed a commit reverting back to the global and it's working --- I must have updated that file after the others.

Should be good now.